tests/io/test_search_iterator_v3.py

```diff
@@ -61,3 +65,19 @@ def test_search_iterator_v3(api):
 
     with pytest.raises(StopIteration):
         next(search_iterator)
+
+
+def test_search_iterator_v3_null_pagination(api):
+    '''
+    Test for null pagination in SearchIterator._process_response
+    '''
+    search_iterator = SearchIterator(
+        api=api
+    )
+    class TempJson:
+        def json(self):
+            return json.loads(json.dumps(NULL_PAGINATION_RESPONSE))
```

Comment:
I dont think we need to test with a full example here:

```suggestion
            return json.loads(json.dumps({'findings': [{'id': 'abcdef'}],
                                          'pagination': None
                                          })
                              )
```